CHANGES.md

```diff
@@ -6,6 +6,8 @@
   * Displays routers in the order that they are defined
 * Update Marathon namer to evaluate an app's running state
 * Add `roundRobin` as a load balancer option.
+* Add `io.l5d.admin` telemeter as an eventual replacement for the `io.l5d.commonMetrics`
```

Comment:
I'm not really a fan of the name _io.l5d.admin_ -- "the admin telemeter" doesn't really tell me much about what it is or does.  Is the intent here "the thing that exports stats on the admin page?"  perhaps `...adminMetricsExport`?